virtcontainers/clh.go

```diff
@@ -120,6 +120,7 @@ var clhKernelParams = []Param{
 	{"no_timer_check", ""}, // do not check broken timer IRQ resources
 	{"noreplace-smp", ""},  // do not replace SMP instructions
 	{"agent.log_vport", fmt.Sprintf("%d", vSockLogsPort)}, // tell the agent where to send the logs
+	{"rootflags", "data=ordered,errors=remount -ro ro"},   // mount the root filesystem as readonly
```

Comment:
typo: `remount -ro` should be `remount-ro`.

Reply:
@sameo Thanks for correcting me. Just being updated.

Reply:
@sameo Just separate the two commits. Any other comments are appreciated. Thanks.